src/www/ui/ui-view-info.php

```diff
@@ -618,58 +618,75 @@ function ShowReportInfo($Upload)
 
     $VT .= "<form action='' name='formReportInfo' method='post'>";
     $VT .= "<table border=1 width='50%' >\n";
+    $withTrStart = "<tr><td align='left'>";
+    $withTdStart = "</td><td align='left'>";
+    $WithTdTrEnd = "</td></tr>";
```

Comment:
Please follow the camleCase coding standard.

Reply:
Nooooo Missed the third one :)

Reply:
Changes done. Remove needs code review label if the review is done.